MunishaKaurBoora_Specialisation_HW2.md

```diff
@@ -0,0 +1,67 @@
+# HOMEWORK WEEK 2 - Revision Game App
+
```

Comment:
I think this is a good idea to have a revision / quiz type of app to test students knowledge , it's always good to have . 
I'm excited to see the final outcome ! ;)